src/additional/mobileRU.js

```diff
@@ -0,0 +1,6 @@
+/*  mobile number for Russia Fedaration */
+$.validator.addMethod( "mobileRU", function( phone_number, element ) {
+	phone_number = phone_number.replace( /\(|\)|\s+|-/g, "" );
+	return this.optional( element ) || phone_number.length > 9 &&
+		phone_number.match( /^((\+7|7|8)+([0-9]){10})$/ );
```

Comment:
Please use `regex.test` instead of `string.match`. We only need to know whether the string matches the regex or not.
`phone_number.match( /^((\+7|7|8)+([0-9]){10})$/ )` => `/^((\+7|7|8)+([0-9]){10})$/.test( phone_number )`